src/invoices/PrintVoucher/PrintVoucherContainer.js

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { useReactToPrint } from 'react-to-print';
+
+import {
+  ConfirmationModal,
+  Loading,
+} from '@folio/stripes/components';
+
+import usePrintData from './usePrintData';
+import PrintContent from './PrintContent';
+
+const PrintVoucherContainer = ({ invoice, closePrint }) => {
+  const componentRef = useRef();
+  const handlePrint = useReactToPrint({
+    content: () => componentRef.current,
+    onAfterPrint: closePrint,
+  });
+
+  const { dataSource, isLoading } = usePrintData(invoice);
+
+  return (
+    <>
+      <ConfirmationModal
+        heading=""
+        message={isLoading ? Loading : 'Print voucher?'}
```

Comment:
translate?

Reply:
I think it's an extra work, it will be replaced with some select fields modal, so no need to introduce temporary strings to translate